lib/puppet/type/java_ks.rb

```diff
@@ -59,50 +58,48 @@ def insync?(is)
     end
 
     newparam(:target) do
-      desc 'Destination file for the keystore.  We autorequire the parent
-        directory for convenience.'
+      desc 'Destination file for the keystore.  This is an autorequire.'
```

Comment:
This doesn't make sense. Needs to include that the parent directory is what is being auto-required.